flow/flow.go

```diff
@@ -309,28 +310,43 @@ func (p *Packet) TransportFlow() (gopacket.Flow, error) {
 	return layer.TransportFlow(), nil
 }
 
+//Hash flow with custom func
+func Hash(f gopacket.Flow, hasher hash.Hash) {
```

Comment:
on side note, you should propose a PR to gopacket to use xxHash  : github.com/google/gopacket/flows.go

Reply:
will do, thanks;)